pyproject.toml

```diff
@@ -14,17 +14,6 @@ Source = "https://github.com/data-apis/array-api/"
 Documentation = "https://data-apis.org/array-api/"
 Homepage = "https://data-apis.org/"
 
-[project.optional-dependencies]
-doc = [
-    "sphinx==4.3.0",
-    "sphinx-material==0.0.30",
-    "myst-parser",
-    "sphinx_markdown_tables",
-    "sphinx_copybutton",
-    "docutils<0.18",
-    "sphinx-math-dollar",
-]
```

Comment:
Yep, this is unfortunately needed because `pip` is still not able to install from this dependency list without also installing the project itself.